packages/react/slider/src/Slider.tsx

```diff
@@ -79,11 +80,13 @@ const Slider = React.forwardRef((props, forwardedRef) => {
     onValueChange = () => {},
     ...sliderProps
   } = props;
-  const sliderRef = React.useRef<HTMLSpanElement>(null);
-  const composedRefs = useComposedRefs(forwardedRef, sliderRef);
+  const [slider, setSlider] = React.useState<HTMLSpanElement | null>(null);
+  const composedRefs = useComposedRefs(forwardedRef, (node) => setSlider(node));
   const thumbRefs = React.useRef<SliderContextValue['thumbs']>(new Set());
   const valueIndexToChangeRef = React.useRef<number>(0);
   const isHorizontal = orientation === 'horizontal';
+  // We set this to true by default so that events bubble to forms without JS (SSR)
+  const isFormControl = slider ? slider.closest('form') : true;
```

Comment:
I wonder if what I was suggesting above would work here though, cause our `slider` element here is not a regular control right?

Reply:
Haha exactly...